### PR TITLE
Tree view: Highlight dag run on hover

### PR DESCRIPTION
**Pull request by bbovenzi**
Before, when a user hovered over a task instance, we only highlighted the row to show what task it belonged to. This PR also highlights the column to show the dag run. This now creates a "crosshair" to let a user more easily which task instance they are hovering on.

Screen Shot 2022-02-09 at 3 16 01 PM

|Before |After |
--- | --- |
|Feb-09-2022 13-12-14|Feb-09-2022 13-02-35 |

### CODE REVIEW

File: airflow/www/static/js/tree/dagRuns/Bar.jsx [hunk 1]
```diff
@@ -34,53 +34,76 @@ import { callModalDag } from '../../dag';
...
-        label={<DagRunTooltip dagRun={run} />}
-        hasArrow
-        portalProps={{ containerRef }}
-        placement="top"
-        openDelay={100}
+}) => {
+  let highlightHeight = '100%';
+  if (containerRef && containerRef.current) {
+    const table = containerRef.current.getElementsByTagName('table')[0];
+    highlightHeight = table.offsetHeight - 53; // subtract the height of the duration axis labels
```
Comment (ashb) [inline airflow/www/static/js/tree/dagRuns/Bar.jsx]: Hmmm magic numbers. Is there some way we can not hard code the height of another element here?

**Review by ryanahamilton — COMMENTED** [review]
Can the `yarn.lock` update be excluded from this PR?

**Comment by bbovenzi** [comment]
> Can the `yarn.lock` update be excluded from this PR?

Actually, I forgot that `yarn upgrade` doesn't update `package.json` by default. Fixed.

**Review by ashb — COMMENTED** [review]
LGTM

**Review by ryanahamilton — APPROVED** [review]

**Pull request merged**